src/CommitmentsV2/SFA.DAS.CommitmentsV2.Jobs/ScheduledJobs/OverlappingTrainingDateRequestNotificationToServiceDeskJob.cs

```diff
@@ -20,7 +20,7 @@ public OverlappingTrainingDateRequestNotificationToServiceDeskJob(IMediator medi
             _logger = logger;
         }
 
-        public async Task Notify([TimerTrigger("0 7 * * 1-5", RunOnStartup = false)] TimerInfo timer)
+        public async Task Notify([TimerTrigger("*/5 * * * *", RunOnStartup = false)] TimerInfo timer)
```

Comment:
make this an environment variable if we would like different values in TEST environments?